week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -68,7 +68,10 @@ function checkCodeIsThere(stringText) {
   
   Hint: Use the corresponding array method to split the array.
 */
-function getTransportModes() { }
+function getTransportModes(name) {
+  let transport = name.slice(1, 3);
+  return transport;
+}
```

Comment:
This isn't entirely correct. The function will only return 2 transport modes because slice(1, 3) means 'start at element 1 and stop before element 3'.
A shorter way of writing this is:
return name.slice(1);